docs/components/custom_components.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: default
+title: Custom Components
+parent: Components
+permalink: /docs/Custom_Components
+---
+
+# Implementing a Custom Component
+
+The components we provide here may not fit your specific needs, and you'll need to write a custom component. Below is an example of how this could be done using React.
+
+## Scenario
+
+Let's say you want to use the Discovery's [Answer finding](https://medium.com/ibm-data-ai/finding-concise-answers-to-questions-in-enterprise-documents-53a865898dbd). Instead of returning _documents_, we want to get _answers_ from within those documents.
```

Comment:
```suggestion
Let's say we want to use Watson Discovery's [Answer finding capability](https://medium.com/ibm-data-ai/finding-concise-answers-to-questions-in-enterprise-documents-53a865898dbd). Instead of returning _documents_, we want to get _answers_ from within those documents.
```

Reply:
This is a great idea..  Could you also consider writing the approach that customers could take to modify the existing search behaviour in terms of entities down the left had side to be a selection that results in an AND condition rather than an OR as is the current state. Consider a customer use case where they have a trained WKS model deployed to understand AIRCRAFT and SERIES entities and they want to bae able to drive search results based on a drop down list of AIRCRAFT = "one or ore selections"  AND SERIES = "one or more selections" .

Reply:
i do think you should stick with "you". (sorry to be contradictory, john) ibm style says, "Use the second-person pronoun _you_ as often as possible."
and we take extra care around references to product names. so rather than making Discovery possessive, and even though the link goes to info about the feature it might not hurt to add a short description. i recommend: "Let's say you want to use the Watson Discovery [answer finding] capability to highlight exact answers in query results"

Reply:
I'm fine with "you"-- I was just wanting the first and second sentence to be consistent. :+1: